src/data/ACTIONS/root/PCT.ts

```diff
@@ -88,6 +88,7 @@ export const PCT = ensureActions({
 		onGcd: true,
 		speedAttribute: Attribute.SPELL_SPEED,
 		castTime: 2300,
+		gcdRecast: 3300,
```

Comment:
Having a gcdRecast on the actions ends up implying that they have an additional cooldown like Phlegma, so they get duplicated on the timeline. These changes should be removed and ABC/NotCasting should treat cooldown as gcdRecast if gcdRecast is not present (other modules have that behavior too, even if it's a little wonky)

Reply:
ah okie that makes sense! I'll do that (I must've been over thinking this a lil)
Thank you!